modules/storage-api/src/testFixtures/java/org/apache/ignite/internal/storage/AbstractMvPartitionStorageConcurrencyTest.java

```diff
@@ -162,4 +217,24 @@ private void cleanup() {
             row = pollForVacuum(HybridTimestamp.MAX_VALUE);
         } while (row != null);
     }
+
+    private enum AddAndCommit {
+        ATOMIC {
+            @Override
+            HybridTimestamp perform(AbstractMvPartitionStorageConcurrencyTest test, @Nullable BinaryRow binaryRow) {
+                HybridTimestamp ts = test.clock.now();
+
+                test.addWriteCommitted(ROW_ID, binaryRow, ts);
+
+                return ts;
+            }
+        },
+        NON_ATOMIC {
+            @Override
+            HybridTimestamp perform(AbstractMvPartitionStorageConcurrencyTest test, @Nullable BinaryRow binaryRow) {
+                return test.addAndCommit(binaryRow);
+            }
+        };
+        abstract HybridTimestamp perform(AbstractMvPartitionStorageConcurrencyTest test, @Nullable BinaryRow binaryRow);
```

Comment:
Blank line above?

Reply:
I wonder why I have a green PMD check...

Reply:
Are you using Gradle?